### PR TITLE
Second hw is done

### PR DESCRIPTION
**Pull request by DmitriyMonarkhov**

### CODE REVIEW

File: src/test/java/com/epam/tc/hw2/DriverManager.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+
+public class DriverManager {
```
Comment (ZeroShadows) [inline src/test/java/com/epam/tc/hw2/DriverManager.java]: I believe, this class should be named as BaseTest

File: src/test/java/com/epam/tc/hw2/ex1/runtest/FirstExerciseRunTest.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+import com.epam.tc.hw2.DriverManager;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class FirstExerciseRunTest extends DriverManager {
```
Comment (ZeroShadows) [inline src/test/java/com/epam/tc/hw2/ex1/runtest/FirstExerciseRunTest.java]: Rename it to FirstExercise

File: src/test/java/com/epam/tc/hw2/ex2/runtest/SecondExerciseRunTest.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+import com.epam.tc.hw2.DriverManager;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class SecondExerciseRunTest extends DriverManager {
```
Comment (ZeroShadows) [inline src/test/java/com/epam/tc/hw2/ex2/runtest/SecondExerciseRunTest.java]: Rename it to SecondExercise

**Review by ZeroShadows — CHANGES_REQUESTED** [review]
Minor comments
Also move all versions to the property section in pom.xml

File: src/test/java/com/epam/tc/hw2/DriverManager.java (hunk 1 above)
Comment (DmitriyMonarkhov) [reply src/test/java/com/epam/tc/hw2/DriverManager.java]: All comments have been corrected.

**Review by ZeroShadows — APPROVED** [review]

**Pull request merged**